src/components/Badges/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { bem } from '../../../utils';
+import { Context } from '../../../constants';
+import styles from './StatusBadge.scss';
+
+export interface Props extends React.HTMLAttributes<HTMLDivElement> {
+    /** Content to be rendered inside the container */
+    children: string;
+    /** The Callout context (e.g. info, critical, success etc. - defaults to info) */
+    context?: Context | 'neutral';
+    /** Icon before text, optional */
+    leadingIcon?: React.ReactElement;
```

Comment:
Nice. I completely missed it because it is not in the design, only in the playground. :-)